python/mleap/sklearn/extensions/data.py

```diff
@@ -61,18 +63,20 @@ def predict(self, X):
 This is because the Imputer both in Spark and MLeap operates on a scalar value and if we were to add a feature extractor in
 front of it, then it would serialize as operating on a tensor and thus, fail at scoring time. 
 """
-class Imputer(SklearnImputer):
+class Imputer(SKLearnImputer):
 
-    def __init__(self, missing_values="NaN", strategy="mean",
-                 axis=0, verbose=0, copy=True, input_features=None, output_features=None):
+    def __init__(self, input_features, output_features,
+                 missing_values=np.nan, strategy="mean",
+                 fill_value=None, verbose=0, copy=True, add_indicator=False):
+        self.op = "imputer"
```

Comment:
i am a bit confused as to how did this work before, was there some place else where this was being done?

Reply:
Oh I don't think this is necessary actually because we call `setattr` on the parent `SKLearnImputer`:
https://github.com/combust/mleap/pull/664/files?file-filters%5B%5D=.py&file-filters%5B%5D=.txt#diff-47b26af574b6dfeab149aeaa450acefa3ed9e37daa90407b90a8076ded3a5db4R121